rpc/core/README.md

```diff
@@ -0,0 +1,15 @@
+# Tendermint RPC
+
+## Generate markdown for [Slate](https://github.com/tendermint/slate)
+
+We are using [Slate](https://github.com/lord/slate) to power our RPC
+documentation. If you are changing a comment, make sure to copy the resulting
+changes to the slate repo and make a PR
```

Comment:
Could we update the doc to make it:
to our slate fork `[here](https://github.com/tendermint/slate)` and make a PR `[there](https://github.com/tendermint/slate)`

Otherwise initially I was thinking we need to make the changes to the original slate repo.